build.sbt

```diff
@@ -45,7 +45,8 @@ coverageFailOnMinimum := true
 coverageExcludedPackages := "<empty>;com.kenshoo.play.metrics.*;.*definition.*;prod.*;testOnlyDoNotUseInAppConf.*;app.*;uk.gov.hmrc.BuildInfo"
 
 scalacOptions ++= Seq(
-  "-P:silencer:pathFilters=views;routes"
+  "-Wconf:src=routes/.*:s",
+  "-Wconf:cat=unused-imports&src=views/.*:s"
 )
```

Comment:
As this was updated, please remove [here](https://github.com/hmrc/national-insurance-des-stub/blob/main/project/AppDependencies.scala#L17-L18) and [here](https://github.com/hmrc/national-insurance-des-stub/blob/main/project/AppDependencies.scala#L5-L6). Also, please bump the dependencies. Thanks

Reply:
Also bump coverage min to 99